crates/cli/src/lib.rs

```diff
@@ -69,9 +69,12 @@ impl Args {
             Command::Start { config: config_path } => {
                 let config = CommitBoostConfig::from_file(&config_path);
 
-                if let Some(modules) = config.modules {
-                    let signer_config = config.signer.expect("missing signer config with modules");
+                // TODO: fix with config
+                let pbs_jwt = "MY_PBS_TOKEN";
```

Comment:
I'd move this closer to where it's needed (inside the `else` branch).
As a side question to clarify: this line is quasi "generating" the token, right?